packages/lesswrong/server/databaseIndexes/commentsDbIndexes.ts

```diff
@@ -116,6 +116,16 @@ export function getDbIndexesOnComments() {
     WHERE ("baseScore" >= 15)
   `);
   
+  // Added to optimize the UltraFeed comment query by filtering on universal status checks
+  void indexSet.addCustomPgIndex(`
```

Comment:
Disbelieve that this partial index is doing something useful, since it reduces the number of comments we need to search over by like 3%.  (Also it doesn't even have any fields indexed, so I'm not sure postgres will ever choose to use it if you're actually filtering based on any other fields?)